ledger/ledger-api-test-tool/src/main/scala/com/daml/ledger/api/testtool/suites/ContractIdIT.scala

```diff
@@ -26,171 +27,202 @@ import scala.util.{Failure, Success, Try}
 // - Central committer ledger implementations (sandboxes, KV...) may accept non-suffixed CID
 // - Distributed ledger implementations (e.g. Canton) must reject non-suffixed CID
 final class ContractIdIT extends LedgerTestSuite {
-  private[this] val nonSuffixedV1Cid = (0 to 32).map("%02x".format(_)).mkString
-  private[this] val suffixedV1Cid = (0 to 48).map("%02x".format(_)).mkString
 
-  private[this] def camlCase(s: String) =
-    s.split("[ -]").iterator.map(_.capitalize).mkString("")
-
-  List[(String, String, Boolean, Features => Boolean, String)](
-    (
-      nonSuffixedV1Cid,
-      "non-suffixed V1",
-      true,
-      features => features.contractIds.v1.isNonSuffixed || features.contractIds.v1.isBoth,
-      "non-suffixed V1 contract IDs are not supported",
+  List(
+    TestConfiguration(
+      description = "v0",
+      example = v0Cid,
+      accepted = true,
+      isSupported = features => features.contractIds.v0.isSupported,
+      disabledReason = "V0 contract IDs are not supported",
     ),
-    (
-      nonSuffixedV1Cid,
-      "non-suffixed V1",
-      false,
-      features => !(features.contractIds.v1.isNonSuffixed || features.contractIds.v1.isBoth),
-      "non-suffixed V1 contract IDs are supported",
+    TestConfiguration(
+      description = "v0",
+      example = v0Cid,
+      accepted = false,
+      isSupported = features => features.contractIds.v0.isNotSupported,
+      disabledReason = "V0 contract IDs are supported",
```

Comment:
```suggestion
```

Reply:
I think you meant that the other one should be deleted. This one asserts that v0 contract IDs are rejected.

Reply:
My mistake.